src/i18n/strings/en_EN.json

```diff
@@ -3322,7 +3322,8 @@
     "By approving access for this device, it will have full access to your account.": "By approving access for this device, it will have full access to your account.",
     "Scan the QR code below with your device that's signed out.": "Scan the QR code below with your device that's signed out.",
     "Start at the sign in screen": "Start at the sign in screen",
-    "Select 'Scan QR code'": "Select 'Scan QR code'",
+    "Select '%(scanQRCode)s'": "Select '%(scanQRCode)s'",
+    "Scan QR code": "Scan QR code",
```

Comment:
Used on Android and iOS projects. Check Weblate how automatic suggestion works, if you are not familiar with it.